### DISCUSSION TITLE
Hourly Load Data

### DISCUSSION THREAD

**Discussion by Mostafa494** [Critical-Infrastructure-Systems-Lab/PowNet#6] [Q&A]
Dear Dr. Kamal,
Thank you for your help.
Actually I do not have hourly load data for each node (bus), So Is there is a way that I could define a hourly load pattern as a total and not for each node ( as I have a lack of data in this area for excel sheet data_load).
Regards

**Comment by kamal0013** [comment]
Yes, if you have demand data for just only one node, you can define that as a gd_node (if it includes generator) or td_node (if not), while the rest can be gn_nodes and tn_nodes.
